ow_plexil/src/plexil-adapter/CMakeLists.txt

```
@@ -6,13 +6,17 @@ set (HEADERS
   OwAdapter.h
   joint_support.h
   subscriber.h
+  PlexilPlanSelection.h
+  TerminalPlanSelection.h
 )
 
 set (SOURCES
   OwExecutive.cpp
   OwInterface.cpp
   OwAdapter.cpp
   subscriber.cpp
+  PlexilPlanSelection.cpp
+  TerminalPlanSelection.cpp
```

File names should be according [ROS C++ Style Guide](http://wiki.ros.org/CppStyleGuide#Files) 
```suggestion
  plexil_plan_selection.cpp
  terminal_plan_selection.cpp
```

---

I take this point back